command.go

```diff
@@ -120,23 +122,35 @@ func (cmd *CommandFunc) configure() {
 		panic("cli.Command: expected a function as argument but got " + t.String())
 	}
 
+	ctx := reflect.TypeOf((*context.Context)(nil)).Elem()
```

Comment:
```suggestion
	ctxType := reflect.TypeOf((*context.Context)(nil)).Elem()
```
Just naming it `ctx` is a bit misleading this that's often the name of an actual context value.